devo/api/client.py

```diff
@@ -29,18 +29,23 @@ class Client(object):
     """
     The Devo SERach REst Api main class
     """
+    app_name = 'python-sdk-app'
```

Comment:
This is a class variable, an attribute of the future objects. These class variables must be in uppercase characters?

Reply:
It is external to the class and can be used from anywhere (both from the class and by from an import *), therefore, the name must be very specific, and capitalized, to avoid overwriting on our own code or by a third party code